smithy-aws-protocol-tests/model/awsJson1_0/required.smithy

```diff
@@ -0,0 +1,125 @@
+$version: "2.0"
+
+namespace aws.protocoltests.json10
+
+use aws.protocols#awsJson1_0
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+apply OperationWithDefaults @httpResponseTests([
+    {
+        id: "AwsJson10ClientErrorCorrectsWhenServerFailsToSerializeRequiredValues",
+        documentation: "Client error corrects when server fails to serialize required values.",
+        protocol: awsJson1_0,
+        code: 200,
+        bodyMediaType: "application/json",
+        headers: {
+            "Content-Type": "application/x-amz-json-1.0",
+        },
+        body: "{}",
+        params: {
+            "requiredString": "",
+            "requiredBoolean": false,
+            "requiredList": [],
+            "requiredDocument": null,
+            "requiredTimestamp": 0,
+            "requiredBlob": "",
+            "requiredByte": 0,
+            "requiredShort": 0,
+            "requiredInteger": 0,
+            "requiredLong": 0,
+            "requiredFloat": 0,
+            "requiredDouble": 0,
+            "requiredMap": {},
+            "requiredBigInteger": 0,
+            "requiredBigDecimal": 0,
+            "requiredEnum": "UNKNOWN"
```

Comment:
Does this not fail validation of the test case? Also, we probably want a more distinct placeholder for this if we're doing this in tests. Something like `__UNKNOWN__`?